.circleci/config.yml

```diff
@@ -262,18 +262,69 @@ jobs:
           env: local
   integration-tests-alpha:
     executor: puppeteer
-    parallelism: 1 # No way to prevent multiple Slack messages otherwise
+    parallelism: 4
     steps:
       - integration-tests:
           env: alpha
-      - notify-qa
   integration-tests-staging:
     executor: puppeteer
-    parallelism: 1 # No way to prevent multiple Slack messages otherwise
+    parallelism: 4
     steps:
       - integration-tests:
           env: staging
-      - notify-qa
+  notify-test-results:
```

Comment:
`notify-test-results`: Waits for test job to finish by polling job status. Finally, calls `./slack/slack-circleci-results.js` to collect all tests summaries, find any failed tests, then send Slack notifications.